suites/apis/rasIntegrationTest.js

```diff
@@ -115,11 +115,48 @@ Scenario('Use client creds from RAS Test User 1 and auth code to obtain access t
 
   const tokens = JSON.parse(obtainTokensForRASUser1);
 
+  // store refresh token to be used in the next test scenario
+  I.cache.rasUser1RefreshToken = tokens.refresh_token;
+
   // decode JWT / Access Token
   const accessTokenJson = parseJwt(tokens.access_token);
 
-  console.log(`access token scopes: ${accessTokenJson.aud}`);
+  // PXP-6552 - Also decode the id_token and check if the client id is in place
+  const idTokenJson = parseJwt(tokens.id_token);
+  expect(idTokenJson).to.have.property('aud');
+  expect(idTokenJson.aud).to.include(I.cache.rasUser1ClientId);
+
+  // run curl with access token and assert the claim is in place
+  const userInfoOutputCmd = `curl -H "Authorization: bearer ${tokens.access_token}" https://${process.env.HOSTNAME}/user/user`;
+  const userInfoOutput = bash.runCommand(userInfoOutputCmd);
+  console.log(`userInfoOutput: ${JSON.stringify(userInfoOutput.ga4gh_passport_v1)}`);
+  const userInfoOutputJson = JSON.parse(userInfoOutput);
+
+  expect(userInfoOutputJson).to.have.property('ga4gh_passport_v1');
+  expect(userInfoOutputJson.ga4gh_passport_v1).to.not.to.be.empty;
+
+  console.log(`access token audience: ${accessTokenJson.aud}`);
+  console.log(`access token scope: ${accessTokenJson.scope}`);
+  expect(accessTokenJson.aud).to.include('ga4gh_passport_v1');
+  expect(accessTokenJson.scope).to.include('ga4gh_passport_v1');
+});
+
+Scenario('Refresh the access token with the refresh_token obtained through the OIDC bootstrapping for RAS Test User 1 @rasAuthN', async (I) => {
+  const refreshAccessTokenCmd = `curl --user "${I.cache.rasUser1ClientId}:${I.cache.rasUser1SecretId}" -X POST "https://${process.env.HOSTNAME}/user/oauth2/token?grant_type=refresh_token&refresh_token=${I.cache.rasUser1RefreshToken}&client_id=${I.cache.rasUser1ClientId}&client_secret=${I.cache.rasUser1SecretId}"`;
+
+  const refreshAccessTokenOutput = bash.runCommand(refreshAccessTokenCmd);
+  console.log(`refreshAccessTokenOutput: ${refreshAccessTokenOutput}`);
+
+  const refreshAccessTokenOutputJson = JSON.parse(refreshAccessTokenOutput);
+
+  expect(refreshAccessTokenOutputJson).to.have.property('access_token');
+  expect(refreshAccessTokenOutputJson).to.have.property('id_token');
+  expect(refreshAccessTokenOutputJson).to.have.property('refresh_token');
+
+  // decode JWT / Access Token
+  const accessTokenJson = parseJwt(refreshAccessTokenOutputJson.access_token);
   expect(accessTokenJson.aud).to.include('ga4gh_passport_v1');
```

Comment:
Yeah feel free to remove this now or else we'll just have to remember to remove it should we ever remove scope from aud